src/BotwinExtensions.cs

```diff
@@ -12,6 +12,10 @@ namespace Botwin
 
     public static class BotwinExtensions
     {
+        /// <summary>
+        /// Adds Botwin to the specified Microsoft.AspNetCore.Builder.IApplicationBuilder with the specified Microsoft.AspNetCore.Routing.IRouter .
```

Comment:
`Adds Botwin to the specified <see cref="IApplicationBuilder"/>`

Reply:
@jchannon thanks! I wrapped up the comments for the public methods. Lemme know what you think.